.env.example

```diff
@@ -9,3 +9,10 @@ OPENAI_API_KEY=""
 # Optional but required if send_email action is used
 RESEND_API_KEY=""
 RESEND_EMAIL=""
+
+# Posthog
+ADMYRAL_POSTHOG_API_KEY="phc_RIpkRea4KLW6EONEDCSZVR1Td4YzeHf4ziUsGzmPnjD"
+ADMYRAL_POSTHOG_HOST="https://eu.i.posthog.com"
```

Comment:
no need to put the posthog credentials in `.env.example` because it doesn't matter for development.